src/pages/VenuePage/VenuePage.tsx

```diff
@@ -209,18 +208,17 @@ export const VenuePage: React.FC = () => {
   // if (isAccessDenied) {
   //   return <AccessDeniedModal venueId={venueId} venueName={venue.name} />;
   // }
+  const { entrance, template, hasPaidEvents } = venue;
+  const { enteredVenueIds } = profile;
+
+  const hasEntrance = Array.isArray(entrance) && entrance.length > 0;
```

Comment:
Entrance step will now only work if the field in DB is an array with at least one element.